docs/docs/administration/single-sign-on.md

```diff
@@ -0,0 +1,244 @@
+| [Auth0](https://auth0.com/docs/get-started/auth0-overview/create-applications) | `auth0` | [Kakao](https://developers.kakao.com/docs/latest/en/kakaologin/utilize#oidc) | `kakao` | [Microsoft](https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc) | `microsoft` |
+| [GitLab](https://docs.gitlab.com/integration/openid_connect_provider/) | `gitlab` | [Keycloak](https://www.keycloak.org/securing-apps/oidc-layers) | `keycloak` | [Okta](https://developer.okta.com/docs/guides/sign-into-web-app-redirect/asp-net-core-3/main/#create-an-app-integration-in-the-admin-console) | `okta` |
+
+If you would like to use a different provider, find your IdP on [this list](https://docs.allauth.org/en/latest/socialaccount/providers/index.html) and navigate to its page. The key is the part of the "development callback URL" between `accounts/` and `/login`.
+
+#### 1b. Create the Mathesar application in your IdP
+
+Create an OAuth application within your IdP using the process you usually follow.
+
+During this process, you'll be asked to specify a "Callback URL" (also called a Redirect URI or Login URL, depending on the provider). This is the URL your IdP uses to return users to Mathesar after a successful login.
+
+In your provider's settings, set the Callback URL to:
+
+```
+https://[YOUR MATHESAR DOMAIN]/auth/oidc/[IDP KEY FROM 1a]/login/callback/
+```
+
+!!! warning ""
+	Ensure you replace the two variables in the example URL above. Your SSO configuration will not work unless this URL is an exact match.
+
+### 2. Enable SSO in Mathesar
+
+Enabling SSO in Mathesar requires setting up a new configuration file or environment variable to hold the necessary IdP-related settings.
+
+#### 2a. Create configuration file.
+
+You'll either create a file named `sso.yml` or set up the `OIDC_CONFIG_DICT` environmental variable. Which you create, and where you save it depends on how you installed Mathesar:
+
+=== "Docker Compose"
+
+    If you used [Docker Compose](./install-via-docker-compose.md), create the `sso.yml` file next to your [`docker-compose.yml` file](https://github.com/mathesar-foundation/mathesar/raw/{{mathesar_version}}/docker-compose.yml):
+
+    ```diff
+    mathesar
+     ├── docker-compose.yml
+     ├── msar/
+    +└── sso.yml
+    ```
+
+    Then uncomment the following lines in your docker compose file:
+
+    ```diff
+     volumes:
+       - ./msar/static:/code/static
+       - ./msar/media:/code/media
+     # Uncomment the following to mount sso.yml and enable Single Sign-On (SSO).
+    -# - ./sso.yml:/code/sso.yml
+    +  - ./sso.yml:/code/sso.yml
+    ```
+
+=== "Directly on Linux, macOS, or WSL"
+
+    For [direct installations](./install-from-scratch.md), create `sso.yml` in the installation directory you [defined while installing](./install-from-scratch.md#set-up-your-installation-directory) Mathesar.
+
+=== "Other install methods"
```

Comment:
Can we make this consistent with the "Activate SSO" section? It mentions "For DigitalOcean or Railway" instead of "Other install methods" there.

Reply:
will do. that's also important because the tab groups are linked.